packages/react/src/app-layout/AppLayoutSidebarNav.tsx

```diff
@@ -186,12 +294,21 @@ function AppLayoutSidebarNavItemInner({
 					alignItems: 'center',
 
 					gap: mapSpacing(0.75),
-					color: boxPalette[isActive ? 'foregroundText' : 'foregroundAction'],
-					...(isActive && {
-						fontWeight: tokens.fontWeight.bold,
+					color:
+						(isActive && level !== 2) || isCurrentPage
```

Comment:
Not sure if `level !== 2` is the right check? `level === 1` might be better (and consistent with L282), but neither are totally future-proof

Reply:
I think the best future for this component is a breaking change or an "AppLayoutNext". It's already leaving button based nav behind and every added feature is nursing so much legacy that the complexity is exponential.